tests/shouldwork/TopEntity/PortGeneration.hs

```diff
@@ -55,7 +53,7 @@ data Passthrough a b = Passthrough a b
 topEntity1 :: "in1" ::: Signal System Int
            -> "in2" ::: Signal System Bool
            -> "out" ::: Signal System Int
-topEntity1 = undefined
+topEntity1 _ _ = pure (deepErrorX "out")
```

Comment:
We need this change, and the other changes in this file, for the same reasons as we need them for `tests/shouldwork/Basic/T1591.hs`